mw-webapp/src/dataAccessLogic/DTOToPreviewConverter/wayDTOToWayPreviewConverter.ts

```diff
@@ -2,8 +2,17 @@ import {UserPreview} from "src/model/businessModelPreview/UserPreview";
 import {WayPreview} from "src/model/businessModelPreview/WayPreview";
 import {WayDTO} from "src/model/firebaseCollection/WayDTO";
 
+/**
+ * Way prps
+ */
 interface WayProps {
+  /**
+   * Owner
```

Comment:
Did you just repwrrite prop name when you added a comment? I don't see the desire to help to read for other devs with such docs. Please,stop and think next time

Reply:
Changed